extensions/braze/braze-currents-users-event.schema.json

```diff
@@ -0,0 +1,300 @@
+                  "enum": [
+                    "true"
+                  ],
+                  "meta:enum": {
+                    "true": "true"
+                  }
+                },
+                "braze:isAMP": {
+                  "title": "Is AMP",
+                  "type": "boolean",
+                  "description": "Indicates that this is an AMP event."
+                },
+                "braze:URL": {
+                  "title": "URL",
+                  "type": "string",
+                  "description": "URL that the user clicked on."
+                },
+                "braze:linkID": {
+                  "title": "Link ID",
+                  "type": "string",
+                  "description": "Unique ID for the link which was clicked, as created by Braze."
+                },
+                "braze:linkAlias": {
+                  "title": "Link Alias",
+                  "type": "string",
+                  "description": "Alias associated with this link ID."
+                },
+                "braze:bounceReason": {
+                  "title": "Bounce Reason",
+                  "type": "string",
+                  "description": "The SMTP reason code and user friendly message received for this bounce event."
+                },
+                "braze:isDrop": {
+                  "title": "Is Drop",
+                  "type": "boolean",
+                  "description": "Indicates that this event counts as a drop event."
+                }
+              }
+            }
+          }
+        },
+        "braze:adTracking": {
+          "title": "Ad Tracking",
+          "type": "object",
+          "properties": {
+            "braze:adID": {
+              "title": "Ad ID",
+              "type": "string",
+              "description": "Advertising identifier."
+            },
+            "braze:adIDType": {
+              "title": "Ad ID Type",
+              "type": "string",
+              "description": "One of 'ios_idfa', 'google_ad_id', 'windows_ad_id', OR 'roku_ad_id'.",
+              "enum": [
+                "ios_idfa",
+                "google_ad_id",
+                "windows_ad_id",
+                "roku_ad_id"
+              ],
+              "meta:enum": {
+                "ios_idfa": "iOS ID for advertisers",
+                "google_ad_id": "Google Ad ID",
+                "windows_ad_id": "Windows Advertiser ID",
+                "roku_ad_id": "Roku Ad ID"
+              }
+            },
+            "braze:adTrackingEnabled": {
+              "title": "Ad Tracking Enabled",
+              "type": "boolean",
+              "description": "Whether advertising tracking is enabled for the device."
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/currents-users-event"
+    }
+  ],
+  "required": [
```

Comment:
Not required, these are already defined as required in the experience event class
https://github.com/adobe/xdm/blob/master/components/classes/experienceevent.schema.json